ts-defs-src/test-api/test-controller.d.ts

```diff
@@ -82,6 +82,20 @@ interface Browser {
     prettyUserAgent: string;
 }
 
+type WindowSelector = unknown;
```

Comment:
I think the `WindowDescriptor` and `WindowDescriptorPromise` are more suitable names.

Reply:
We use it in the code: https://github.com/DevExpress/testcafe/blob/87094b31c74d1ea20b3286f2e07dcaef3d87a5df/src/api/test-controller/index.js#L320

I just wanted to be consistent with the code. I agree that `WindowDescriptor` can be better, but we then should rename corresponding names in the code too. @AlexKamaev, do you agree?

Reply:
Yes, I agree that WindowDescriptor is more suitable and not so confusing as Selector

Reply:
Ok